src/main/java/org/symphonyoss/integration/webhook/salesforce/parser/v2/OpportunityNotificationMetadataParser.java

```diff
@@ -75,4 +80,138 @@ protected void postProcessOutputData(EntityObject output, JsonNode input) {
+
+    if (!StringUtils.isEmpty(amount)) {
+      amount = NumberFormatUtils.formatValueWithLocale(Locale.US, amount);
+    }
+
+    return amount;
+  }
+
+  private void proccessAmountAndCurrencyIsoCode(JsonNode node) {
+    String amountAndCurrencyIsoCodeFormatted = EMPTY;
+
+    String amount = getAmountFormatted(node);
+    if (!StringUtils.isEmpty(amount)) {
+      amountAndCurrencyIsoCodeFormatted = amount;
+    }
+
+    String currencyIsoCode = node.path(SalesforceConstants.CURRENCY_ISO_CODE).asText(EMPTY);
+    if (!StringUtils.isEmpty(currencyIsoCode)) {
+      if (!StringUtils.isEmpty(amountAndCurrencyIsoCodeFormatted)) {
+        amountAndCurrencyIsoCodeFormatted = amountAndCurrencyIsoCodeFormatted + SEPARATOR + currencyIsoCode;
+      } else {
+        amountAndCurrencyIsoCodeFormatted = currencyIsoCode;
+      }
+    }
+
+    ((ObjectNode) node).put(SalesforceConstants.AMOUNT_AND_CURRENCY_ISO_CODE, amountAndCurrencyIsoCodeFormatted);
+  }
+
+  private void processAccountName(JsonNode node) {
+    String accountName = node.path(SalesforceConstants.NAME).asText(EMPTY);
+
+    if (StringUtils.isEmpty(accountName)) {
+      ((ObjectNode) node).put(SalesforceConstants.NAME, DEFAULT_VALUE_NULL);
+    }
+  }
+
+  private boolean emailExistsAtSymphony(String emailAddress) {
+    if (StringUtils.isBlank(emailAddress)) {
+      return false;
+    }
+
+    User user = userService.getUserByEmail(integrationUser, emailAddress);
+    return user.getId() != null;
+  }
+
+  private void proccessCloseDate(JsonNode node) {
+    String closeDateFormat = node.path(SalesforceConstants.CLOSE_DATE).asText(null);
+    SimpleDateFormat formatter = new SimpleDateFormat(SalesforceConstants.TIMESTAMP_FORMAT);
+
+    if (!StringUtils.isEmpty(closeDateFormat)) {
+      try {
+        closeDateFormat = formatter.format(formatter.parse(closeDateFormat));
+      } catch (ParseException e) {
+        // Do nothing
+      }
+
+      ((ObjectNode) node).put(SalesforceConstants.CLOSE_DATE, closeDateFormat);
+    }
+  }
+
+  private void proccessURLIconIntegration(JsonNode node) {
+    String urlIconIntegration = getURLFromIcon("salesforce.svg");
+
+    if (!urlIconIntegration.isEmpty()) {
+      ((ObjectNode) node).put(SalesforceConstants.URL_ICON_INTEGRATION, urlIconIntegration);
+    }
+  }
+
+  private void processUpdatedFields(JsonNode currentNode, JsonNode previousNode) {
+    String updatedFields = null;
+
+    Iterator<Map.Entry<String, JsonNode>> fields = previousNode.fields();
+    while (fields.hasNext()) {
+
+      String fieldKey = fields.next().getKey();
+
+      if (!StringUtils.isBlank(SalesforceConstants.getOpportunityFieldName(fieldKey))) {
+        if (StringUtils.isEmpty(updatedFields)) {
+          updatedFields = SalesforceConstants.getOpportunityFieldName(fieldKey);
+        } else {
+          updatedFields = updatedFields + ", " + SalesforceConstants.getOpportunityFieldName(fieldKey);
```

Comment:
You could use a constant here, using same strategy you have used with other literals.